### PR TITLE
shaky minigame + i removed the bg color change in the rhythm game lmao

### PR DESCRIPTION
**Pull request by ItsRaijin**
c:

### CODE REVIEW

File: src/controller.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ import {
 } from "./interlude/interlude";
 import { CheckpointMinigame } from "./minigames/checkpointMinigame";
 import { ScrubMinigame } from "./minigames/scrubMinigame";
+import { ShakingMinigame } from "./minigames/shakingMinigame";
```
Comment (zainafzal08) [inline src/controller.ts:16]: Do you need to add this to ALL_MINIGAMES?

File: src/minigames/shakingMinigame.ts [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  Texture,
+  Assets,
+  Point,
+  Text,
+  Sprite,
+  TextStyle,
+} from "pixi.js";
+import { Minigame } from "./minigame";
+
+export class ShakingMinigame extends Minigame{
```
Comment (zainafzal08) [inline src/minigames/shakingMinigame.ts:13]: Set lifetime to like 30_000, the default 10 second timeout is too short for this i think

**Review by zainafzal08 — APPROVED** [review]

File: src/controller.ts (hunk 1 above)
Comment (josephkimsungha) [reply src/controller.ts:16]: (hint: it's now called MINIGAMES_POOL)

File: src/minigames/shakingMinigame.ts [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    }
+
+    private onDrag(event: FederatedMouseEvent) {
+        if (!this.dragObject) return;
+
+        const previousPos = this.dragObject.position.clone();
+        this.dragObject.position = event.getLocalPosition(this.dragObject.parent);
+
+        this.vel = this.dragObject.position.subtract(previousPos);
+        this.score += Math.floor((Math.abs(this.vel.x) + Math.abs(this.vel.y))/10);
```
Comment (josephkimsungha) [inline src/minigames/shakingMinigame.ts:78]: I thought we wanted to make the score increase based on change in velocity? But we can land this first and update later

**Review by josephkimsungha — APPROVED** [review]

File: src/minigames/shakingMinigame.ts (hunk 2 above)
Comment (ItsRaijin) [reply src/minigames/shakingMinigame.ts:13]: when playing i could shake it within 6 seconds

**Pull request merged**